native/src/rapidsml_jni.cu

```diff
@@ -75,6 +103,17 @@ cublasOperation_t convertToCublasOpEnum(int int_type)
       break;
   }
 }
+
+// TODO: optimize for GPU
```

Comment:
Oh, this just reminds me that leveraging `CUBLAS_OP_T` properly and swap the position of A&B can help get rid of the extra transpose. Because when matrix: `A * B = C` , we can get `B(t) * A(t) = C(t)` (t means transpose). I add more comments in the UDF part.